src/acl.c

```diff
@@ -1041,6 +1043,13 @@ unsigned long ACLGetCommandID(const char *cmdname) {
     return thisid;
 }
 
+/* Clear command id table and reset nextid to 0. */
+void ACLClearCommandID(void) {
+    if (commandId) raxFree(commandId);
+    commandId = NULL;
+    nextid = 0;
+}
```

Comment:
why is this needed? i see only called on startup.

Reply:
Hi @oranagra, the reason of using this function is in current code logic, populateCommandTable function has been called earlier, and sentinel is replacing with a different command table later in initSentinel call, so it need to clear the previous assignment of the command ids. Providing this function will make code clear and safe rather than directly changing in main server logic (populateCommandTable function).